utils/src/main/java/com/redhat/lightblue/migrator/utils/DAOFacadeBase.java

```diff
@@ -31,12 +33,40 @@
 
     protected final D legacyDAO, lightblueDAO;
 
+    private EntityIdStore entityIdStore = null;
+
+    public EntityIdStore getEntityIdStore() {
+        return entityIdStore;
+    }
+
+    public void setEntityIdStore(EntityIdStore entityIdStore) {
+        this.entityIdStore = entityIdStore;
+
+        try {
+            Method method = lightblueDAO.getClass().getMethod("setEntityIdStore", EntityIdStore.class);
+            method.invoke(lightblueDAO, entityIdStore);
```

Comment:
It does not require a change in the dao interface. Your lightblueDAO implementation needs to have that setter, but it's a net new class.